test/e2e/minscale_readiness_test.go

```diff
@@ -156,17 +205,42 @@ func serverlessServicesName(t *testing.T, clients *test.Clients, revisionName st
 	}); err != nil {
 		t.Fatalf("Error retrieving sks %q: %v", revisionName, err)
 	}
+
 	return privateServiceName
 }
 
-func waitForDesiredScale(t *testing.T, clients *test.Clients, privateServiceName string, cond func(int) bool) error {
+func waitForDesiredScale(t *testing.T, clients *test.Clients, serviceName string, cond func(int) bool) error {
 	endpoints := clients.KubeClient.Kube.CoreV1().Endpoints(test.ServingNamespace)
 
 	return wait.PollImmediate(time.Second, 1*time.Minute, func() (bool, error) {
```

Comment:
I think we need to poll as fast as we can here. With 1s, it is possible to miss the scale down-and-up

Reply:
Switching to 250ms

Reply:
I'd go as low as 50ms